lib/gollum/views/helpers.rb

```diff
@@ -2,31 +2,63 @@
 
 module Precious
   module Views
+    module RouteHelpers
+      ROUTES = {
+        'gollum' => {
+          :last_commit_info => 'last-commit-info',
+          :latest_changes => 'latest_changes',
+          :upload_file => 'uploadFile',
```

Comment:
Similar kind of question here: why `uploadFile` and not `upload_file` or `upload-file`? It seems we should be consistent in our style.